pyvista/core/utilities/features.py

```diff
@@ -247,6 +247,34 @@ def cartesian_to_spherical(x, y, z):
     return r, theta, phi
 
 
+def spherical_to_cartesian(r, theta, phi):
+    """Convert Spherical coordinates to 3D Cartesian coordinates.
+
+    Parameters
+    ----------
+    r : numpy.ndarray
+        Radial distance.
+
+    theta : numpy.ndarray
+        Angle (radians) with respect to the polar axis. Also known
```

Comment:
vtk defines theta and phi the other way around.  I think we should align to how vtk operates.

See https://vtk.org/doc/nightly/html/classvtkSphericalTransform.html#details

#4965 and `pyvista.Sphere` are already consistent with this vtk operation.